Extensions/Multiplayer/peerJsHelper.ts

```diff
@@ -306,17 +333,22 @@ namespace gdjs {
       });
       peer.on('close', () => {
         peer = null;
-        loadPeerJS();
+        initializePeerJS(initOptions);
       });
       peer.on('disconnected', peer.reconnect);
     };
+    export const useDefaultBrokerServer = initializePeerJS;
 
     /**
      * Connects to another p2p client.
      * @param id - The other client's ID.
      */
     export const connect = (id: string) => {
       if (peer === null) return;
+      if (!ready) {
+        _peerIdToConnectToOnceReady = id;
```

Comment:
should this be saved also in case peer=null ? Or is it a case that should not happen

Reply:
Yes, good idea I think